Assets/Scripts/Minigames/CowardMiniGame/RipcordBehavior.cs

```diff
@@ -34,28 +37,56 @@ public class RipcordBehavior : MonoBehaviour
     private int _numReleased;
     private GameObject _preview;
 
+
+
+    private bool _gameStarted;
+
+    [Header("Light")]
+    private int _currentLight;
+    [SerializeField] private GameObject[] _listLights;
+    private float _lightSpeed;
+    private bool _lightDoOnce;
+
     [SerializeField] private GameObject _gears;
 
     private PlayerController _pc;
 
     // Start is called before the first frame update
     void Start()
     {
+        _switchMove = -1;
```

Comment:
These variables can be defined and declared in one line, or maybe make a function that sets these values so start is easier to read